skeleton/webpack/package.json

```diff
@@ -67,6 +71,7 @@
     "json-loader": "",
     "html-loader": "",
     "istanbul-instrumenter-loader": "",
+    "open": "",
```

Comment:
You probably accidentally merged this line back in. This is removed from current aurelia-cli master, "open" has been moved to cli dependencies for better backwards compatibility.

Reply:
So sorry, I fixed it.